fs/vfs/fs_fcntl.c

```diff
@@ -350,11 +350,11 @@ int fcntl(int fd, int cmd, ...)
 
           if (oflags & FD_CLOEXEC)
             {
-              ret = file_ioctl(filep, FIOCLEX, NULL);
+              ret = ioctl(fd, FIOCLEX, NULL);
             }
           else
             {
-              ret = file_ioctl(filep, FIONCLEX, NULL);
+              ret = ioctl(fd, FIONCLEX, NULL);
```

Comment:
let's:

1. move F_DUPFD and F_DUPFD_CLOEXEC to file_vfcntl
2. move file_get/file_put into default block